submissions/edehJohn/html/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
```

Comment:
@john84edeh you already have a product page in the introToHtml. 
You can just update the file if needed. You need to remove this file from the htmlforms folder.